15: [FEATURE] Create model package

skydiverGM opened the pull request:
- Created package 'model' inside the common module
- Added abstract class 'AbstractModel' with common fields
- Included 'id' of Long type, 'createdAt' and 'modifiedAt' of LocalDateTime type

## Describe your changes

## Issue ticket number and link

## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.

ownerofglory reviewed (changes requested):
Looks OK. But created class is not really usable as it contains neither accessor-methods nor constructors. 
Please at least add accessor methods. 
You may as well do the story with [lombok support](https://github.com/users/ownerofglory/projects/7/views/1?pane=issue&itemId=89601607&issue=ownerofglory%7Chillel-expense-tracker-backend%7C11) within this same PR, that would be great.
Thanks! 😊

skydiverGM commented:
Changes have been made as requested:
- Added lombok dependency to pom.xml
- Added @Getter, @Setter, @NoArgsConstructor, @AllArgsConstructor to AbstractModel
Please review again
